packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -788,6 +796,24 @@ export default abstract class SqlIntegration
     return { results: results.rows, duration };
   }
 
+  getDropUnitsTableQuery(params: DropTableQueryParams): string {
+    // valdidate units table query follows expected name to help
+    // prevent dropping other tables
+    if (!params.tableName.includes(UNITS_TABLE_PREFIX)) {
```

Comment:
`startsWith` instead of `includes`

Reply:
Wait, nevermind.  I guess there could be a schema/db added as well.

Reply:
I'm not sure it will because it will because of the database and schema that can be prepended.